extensions/adobe/experience/campaign/orchestration/experienceevent.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/experience/campaign/orchestration/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "User journey experience event",
+  "type": "object",
+  "description": "Orchestration extension to ExperienceEvent",
+  "definitions": {
+    "journey": {
+      "title": "Journey for users",
+      "type": "object",
+      "description": "Journey created by marketer for its users",
+      "properties": {
+        "@id": {
+          "title": "Journey unique identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "The unique identifier of the journey created by the marketer."
+        }
+      },
+      "required": ["@id"]
+    },
+    "journeyVersion": {
+      "title": "Journey version",
+      "type": "object",
+      "description": "Schema for describing the version of a journey, where version holds the state machine",
+      "properties": {
+        "@id": {
+          "title": "Journey version identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "The unique identifier denoting the version of journey on which the user is active"
+        }
+      },
+      "required": ["@id"]
+    },
+    "action": {
+      "title": "Action on step transition",
+      "type": "object",
+      "description": "Action taken during a step transition happening for an user in state machine",
+      "properties": {
+        "xdm:actionID": {
+          "title": "Action identifier",
+          "type": "string",
+          "format": "uri",
+          "description": "Unique identifier denoting the associated action."
+        },
+        "xdm:type": {
```

Comment:
actionType

Reply:
I didn't changed this because it is causing breaking changes as we removed "type" (not backward compatible). Should I go ahead with this change?